packages/unlock-js/package.json

```diff
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "@uniswap/sdk-core": "4.0.6",
-    "@uniswap/smart-order-router": "3.14.0",
```

Comment:
I had to downgrade since the latest version broke the swap and purchase feature.